conf/celeborn-defaults.conf.template

```diff
@@ -16,14 +16,6 @@
 #
 
 # An Example:
-celeborn.rpc.io.mode                             NIO
-celeborn.rpc.io.clientThreads                    8
-celeborn.rpc.io.serverThreads                    8
-celeborn.rpc.io.numConnectionsPerPeer            2
-
-celeborn.push.io.numConnectionsPerPeer           2
-celeborn.worker.push.io.threads                  8
-
 celeborn.metrics.enabled                         true
 
 celeborn.worker.storage.dirs                     /mnt/disk1,/mnt/disk2,/mnt/disk3,/mnt/disk4,/mnt/disk5
```

Comment:
Could this config add disktype in each disk to tell users follow the pattern?

Reply:
NO. This is a config file. The demo of this configuration is in the README that shows how to set disktype for different dirs.